### DISCUSSION TITLE
Limit number of imported elements

### DISCUSSION THREAD

**Discussion by piotrpog** [engram-design/FeedMe#1013] [Ideas]
This might be useful for debugging. If we work with some large file that takes minutes to process and import data from, we might want to for example import only the first 10 elements to check how it works.

**Comment by kristiansp** [comment]
@piotrpog At least if you do it from the CLI there is a limit parameter which I think does what you want. Maybe it also works if you click the gear icon in your feed overview to get the direct URL, and then add try add a limit parameter to it, but I haven't tested it.

**Reply by gixxy22** [reply]
I can confirm that this works!
